src/components/Form/MucCheckbox.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div
+    class="m-checkboxes__item"
+    @click="clickedCheckbox"
+  >
+    <input
+      class="m-checkboxes__input"
+      name="checkbox"
+      type="checkbox"
+      value="checkbox-value-0"
+    />
+    <label
+      class="m-label m-checkboxes__label"
+      for="checkbox-example-"
+    >
+      {{ label }}
+    </label>
+  </div>
+</template>
+
+<script setup lang="ts">
+const checkBoxValue = defineModel({ default: false });
```

Comment:
In Vue I would prefer to have this a model-value and not an internal represantation. This way I can't change the state of the checkbox from outside.

Reply:
But [defineModel()](https://vuejs.org/guide/components/v-model#basic-usage) can be accessed from the outside like a prop + does not need any emits to be updated. Maybe i understand something wrong.

Reply:
> Starting in Vue 3.4, the recommended approach to achieve this is using the [defineModel()](https://vuejs.org/api/sfc-script-setup#definemodel) macro:

sorry, my bad. js libs sometimes move to fast.

Reply:
It's an awesome feature to save some boilerplate code - very much appreciated ☺